### PR TITLE
#25134 Bundler, Handler, Wrapper working v1

### PR DESCRIPTION
**Pull request by dsilvam**
This PR is the second one including partial work for supporting Push Publishing of Experiments. 
Includes: 
* ExperimentBundler
* ExperimentHandler
* ExperimentWrapper
* Using the proper mapper in the BundleUtil

### CODE REVIEW

File: dotCMS/src/enterprise/java/com/dotcms/enterprise/publishing/remote/bundler/ExperimentBundler.java [hunk 1]
```diff
@@ -228,7 +229,7 @@ private void writeExperiment(final BundleOutput bundleOutput, final Experiment e
 
 		if (!bundleOutput.exists(experimentFileUrl)) {
 			try (final OutputStream outputStream = bundleOutput.addFile(experimentFileUrl)) {
-				BundlerUtil.objectToXML(wrapper, outputStream);
+				BundlerUtil.objectToJSON(wrapper, outputStream);
```
Comment (jdotcms) [inline dotCMS/src/enterprise/java/com/dotcms/enterprise/publishing/remote/bundler/ExperimentBundler.java:232]: good

**Review by victoralfaro-dotcms — APPROVED** [review]

File: dotCMS/src/enterprise/java/com/dotcms/enterprise/publishing/remote/handler/ExperimentHandler.java [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+		final Collection<File> experiments = FileUtil.listFilesRecursively
+				(bundleFolder, new ExperimentBundler().getFileFilter());
+
+        handleExperiments(experiments);
+	}
+
+	private void handleExperiments(final Collection<File> experiments) throws DotPublishingException, DotDataException {
+
+	    if(LicenseUtil.getLevel() < LicenseLevel.PROFESSIONAL.level) {
```
Comment (jdotcms) [inline dotCMS/src/enterprise/java/com/dotcms/enterprise/publishing/remote/handler/ExperimentHandler.java:138]: probably not needed

**Review by jdotcms — APPROVED** [review]

File: dotCMS/src/enterprise/java/com/dotcms/enterprise/publishing/remote/handler/ExperimentHandler.java [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+/**
+ * This handler class is part of the Push Publishing mechanism that deals with Experiments-related information inside a
+ * bundle and saves it in the receiving instance. This class will read and process only the {@link ContentType} data
+ * files.
+ * <p>
+ * In dotCMS, content is defined by “Content Types“. When building a site in dotCMS, CMS Administrators design Content
+ * Types that fit their specific use case or content needs. The easiest way to think about a Content Type is as a
+ * content database that you get to design, manage, permission, query and finally display.
+ *
+ * @author Anibal Gomez
```
Comment (freddyucv) [inline dotCMS/src/enterprise/java/com/dotcms/enterprise/publishing/remote/handler/ExperimentHandler.java:104]: ?? hahahha

**Review by freddyucv — APPROVED** [review]

File: dotCMS/src/enterprise/java/com/dotcms/enterprise/publishing/remote/handler/ExperimentHandler.java (hunk 3 above)
Comment (dsilvam) [reply dotCMS/src/enterprise/java/com/dotcms/enterprise/publishing/remote/handler/ExperimentHandler.java:104]: lol done

**Pull request closed without merging**